domains/emissions/Emissions.API/API/Models/QuantityUnit.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.ComponentModel;
+
+namespace API.Models
+{
+    public enum QuantityUnit
+    {
+        [Description("Wh")]
```

Comment:
Are these descriptions used in serialization?
Otherwise, how much value do they add?

Reply:
Unit is serialized